template.tpl

```diff
@@ -998,24 +998,8 @@ ___TEMPLATE_PARAMETERS___
                 "value": "deviceId"
               },
               {
-                "displayValue": "cookieExpiration",
```

Comment:
This feels like a breaking change right? Should we keep the old config for backward compatibility and pick 
1. If the new config exists, use the new one
2. If the old config exists and new does not, use the old one.

Reply:
Yes, this is a breaking change. And if the template is linked (unlink by manually changing the template), GTM will automatically update to the latest template. 

But I don't think we should keep it backward compatible:
- The old UI didn't work which means that even if customers set cookieOptions via the old UI, it didn't really set the configuration
- It makes UI less clean and confusing with two places to set cookieOptioins
- Hard to maintain by keeping two places

Reply:
Ah interesting. I assumed that the old config was working as expected and we were only adding new options. If the old ones didn't work, then that makes sense.